src/LineageTree/lineageTree.py

```diff
@@ -461,28 +461,47 @@ def normalize_values(v, nodes, range, shift, mult):
                     )
         dwg.save()
 
-    def to_treex(self, sampling=1):
-        """Convert the lineage tree into a treex file."""
-        from treex.tree import Tree
-
-        id_to_tree = {id: Tree() for id in self.nodes}
-        times_to_consider = [t for t, n in self.time_nodes.items() if 0<len(n)]
-        times_to_consider = times_to_consider[::sampling]
-        for t in times_to_consider:
-            for id_mother in self.time_nodes[t]:
-                ids_daughters = self.successor.get(id_mother, [])
-                new_ids_daughters = ids_daughters.copy()
-                for _ in range(sampling-1):
-                    tmp = []
-                    for d in new_ids_daughters:
-                        tmp.extend(self.successor.get(d, [d]))
-                    new_ids_daughters = tmp
-                for daugther in new_ids_daughters: ## For each daughter in the list of daughters
-                    id_to_tree[id_mother].add_subtree(id_to_tree[daugther]) ## Add the Treex daughter as a subtree of the Treex mother   
-        
-        roots = [id_to_tree[id] for id in set(self.successor).difference(self.predecessor)] 
+    def to_treex(self, sampling=1,start=0,finish=10000,many=True):
+            """Convert the lineage tree into a treex file."""
+            """ start/finish refer to first index in the new array times_to_consider"""
+            from treex.tree import Tree
+            start//=sampling
+            # if start<=0:
+            #     print("Bigger Start")
+            #     return False
+            finish//=sampling
+            if finish-start<=0:
+                print("Bigger Finish")
+                return False
```

Comment:
Please change `False` by `None`.

Also remove the print.
Maybe put a warning instead?